partiql-types/src/main/kotlin/org/partiql/value/io/PartiQLValueTextWriter.kt

```diff
@@ -293,22 +293,22 @@ internal class PartiQLValueTextWriter(
 
         override fun visitNullableBag(v: NullableBagValue<*>, ctx: Format?) = when (v.isNull()) {
             true -> "null"
-            else -> visitBag(v.promote(), ctx)
+            false -> visitBag(v.promote(), ctx)
         }
 
         override fun visitNullableList(v: NullableListValue<*>, ctx: Format?) = when (v.isNull()) {
             true -> "null"
-            else -> visitList(v.promote(), ctx)
+            false -> visitList(v.promote(), ctx)
         }
 
         override fun visitNullableSexp(v: NullableSexpValue<*>, ctx: Format?) = when (v.isNull()) {
             true -> "null"
-            else -> visitSexp(v.promote(), ctx)
+            false -> visitSexp(v.promote(), ctx)
```

Comment:
You didn't like the 4-char/4-char alignment 😛

Reply:
I have such a love-hate relationship with Kotlin's exhaustive when statement :)